### DISCUSSION TITLE
Custom shader for Bevy UI

### DISCUSSION THREAD

**Discussion by Tam** [bevyengine/bevy#7803] [Q&A]
Is it possible to replace the shader used in Bevy UI for a custom one? 

I've tried overwriting the contents of `UI_SHADER_HANDLE` but come up against issues importing `bevy_render::view`. I'd also like to add custom parameters (i.e. border width), in a similar way that color and image are passed.

**Comment by ickshonpe** [comment]
You could look at #3991
